### DISCUSSION TITLE
Filtering on Google Sheet as Data source doesn't work

### DISCUSSION THREAD

**Discussion by golfpavat** [Budibase/budibase#8536] [Help] [answered]
I'm trying to use the Google Sheet as a data source which works fine. However, when I'm trying to filter out the data to show on the app it's doesn't work. So I went to the data and try filtering that which also doesn't work.
When I switch back to Budibase DB, it works fine. So there must be something I'm not doing right or it's just a bug.

![image](https://user-images.githubusercontent.com/48328979/200136529-f819aff0-5269-40ec-ae65-f6fa60c5ffde.png)

![image](https://user-images.githubusercontent.com/48328979/200136502-fa52f9c6-6588-42a8-83bd-3d0723c7451f.png)

**Comment by melohagan** [comment, marked as answer]
Hey @golfpavat 

This is a bug; thanks for bringing this to our attention. I've raised an issue here: https://github.com/Budibase/budibase/issues/8557
